legal-api/src/legal_api/models/business.py

```diff
@@ -563,6 +563,23 @@ def get_filing_by_id(cls, business_identifier: int, filing_id: str):
             one_or_none()
         return None if not filing else filing[1]
 
+    @classmethod
+    def is_pending_amalgamating_business(cls, business_identifier: int):
```

Comment:
business_identifier is a string (`BC1234567`)

Reply:
Done.